frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/UnitVmModel.java

```diff
@@ -2602,10 +2602,14 @@ private Set<DisplayType> getSupportedDisplayTypes(int osId, BiosType biosType) {
                 .map(Pair::getSecond)
                 .filter(dt -> dt != DisplayType.none)
                 .filter(dt -> biosType.isOvmf() || dt != DisplayType.bochs)
-                .collect(Collectors.toSet());
+                .collect(Collectors.toCollection(LinkedHashSet::new));
     }
 
     public void initDisplayModels(Set<DisplayType> displayTypes) {
+        if (displayTypes.remove(DisplayType.qxl)) {
```

Comment:
I think there is a better solution. What about setting the default DisplayType directly on line 2621?
Instead of 
```
getDisplayType().setItems(displayTypes, displayTypes.iterator().next());
```
lets use
```
DisplayType selectedDisplayType = behavior.getDefaultDisplayType(displayTypes)
getDisplayType().setItems(displayTypes, selectedDisplayType);
```